dmoj/executors/RUST.py

```diff
@@ -8,74 +8,36 @@
 [package]
 name = "user_submission"
 version = "1.0.0"
+edition = "2021"
 
 [dependencies]
 dmoj = "0.1"
-rand = "0.3"
-"""
-
-CARGO_LOCK = b"""\
-cargo-features = ["strip"]
-
-[root]
-name = "user_submission"
-version = "1.0.0"
-dependencies = [
- "dmoj 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)",
- "rand 0.3.15 (registry+https://github.com/rust-lang/crates.io-index)",
-]
+libc = "0.2"
+rand = "0.8"
+rand_xoshiro = "0.6"
 
 [profile.release]
 strip = "symbols"
+"""
 
-[[package]]
-name = "dmoj"
-version = "0.1.5"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-dependencies = [
- "lazy_static 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
- "libc 0.2.18 (registry+https://github.com/rust-lang/crates.io-index)",
-]
-
-[[package]]
-name = "lazy_static"
-version = "0.2.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-
-[[package]]
-name = "libc"
-version = "0.2.18"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-
-[[package]]
-name = "rand"
-version = "0.3.15"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-dependencies = [
- "libc 0.2.18 (registry+https://github.com/rust-lang/crates.io-index)",
-]
-
-[metadata]
-"checksum dmoj 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)" = "a1f8a155771d562ab98db35ed9b4da482ef178eec293eeb1f6302036100e84f1"
-"checksum lazy_static 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)" = "6abe0ee2e758cd6bc8a2cd56726359007748fbf4128da998b65d0b70f881e19b"
-"checksum libc 0.2.18 (registry+https://github.com/rust-lang/crates.io-index)" = "a51822fc847e7a8101514d1d44e354ba2ffa7d4c194dcab48870740e327cac70"
-"checksum rand 0.3.15 (registry+https://github.com/rust-lang/crates.io-index)" = "022e0636ec2519ddae48154b028864bdce4eaf7d35226ab8e65c611be97b189d"
-"""  # noqa: E501
-
-HELLO_WORLD_PROGRAM = """\
-#[macro_use] extern crate dmoj;
-extern crate rand;
+TEST_PROGRAM = """\
+use dmoj as _;
```

Comment:
Aside from this PR, we should also update the site template for Rust code, helloworld, etc.

Are these `use`s actually necessary for runtimes-docker to pull in the right packages (as opposed to trusting our TOML), or are they just to prove that things worked correctly for CI?

Reply:
They are just to prove that stuff works for CI. We can add a comment, or remove them, or whatever.

Reply:
Let's just leave a comment. Having them seems useful.

Reply:
Should be pushed and rebased.